x-pack/plugins/infra/public/components/logging/log_text_stream/scrollable_log_text_stream_view.tsx

```diff
@@ -109,6 +111,7 @@ export class ScrollableLogTextStreamView extends React.PureComponent<
       return {
         ...prevState,
         items: nextItems,
+        isScrollLocked: false,
       };
     }
```

Comment:
Do we maybe want to set `isScrollLocked` to `false` in the fallthrough case as well? That would also disable the scroll lock when stopping the streaming.

Reply:
Yeah, good point, I've added the fallthrough case. And thanks for the review!